db/schema.rb

```diff
@@ -165,6 +187,7 @@
     t.uuid "host_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.boolean "supported"
```

Comment:
This one should be removed. Not part of the migration.

Reply:
Updated! Sorry :)